File: Assignment28_1.java
import java.util.Scanner;

////////////////////////////////////////////////////////////////////////////
//    Function Name :       Pattern
//    Description :         To get the pattern.
//    Input :               Integer
//    Output :              print the pattern
//    Author :              Raj Samir Jadhav
//    Date :                21/11/2025
////////////////////////////////////////////////////////////////
/// 
class Pattern
{
    
    public void display(int iRow, int iCol)                                 // Method to display the pattern
    {
        for(int i = 1; i <= iRow; i++)                                     // Loop for rows
        {
            for(int j = 1; j <= iCol; j++)                               // Loop for columns
            {
                System.out.print((char)('A' + j - 1) + "\t");           // Print A B C D...
            }
            System.out.println();
        }
    }
}

public class Main
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows and columns:");
        int iValue1 = sobj.nextInt();
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.display(iValue1, iValue2);                                    // Method call
    }
}

File: Assignment28_1.c
#include <stdio.h>

////////////////////////////////////////////////////////////////////////////
//    Function Name :       Pattern
//    Description :         To get the pattern.
//    Input :               Integer
//    Output :              print the pattern
//    Author :              Raj Samir Jadhav
//    Date :                21/11/2025
////////////////////////////////////////////////////////////////

void Pattern(int iRow, int iCol)                              // Function to display the pattern
{
    int i = 0, j = 0;

    for(i = 1; i <= iRow; i++)                               // Loop for rows
    {
        for(j = 1; j <= iCol; j++)                         // Loop for columns
        {
            printf("%c\t", 'A' + (j - 1));                 // Print A, B, C, D...
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows and columns: ");
    scanf("%d %d", &iValue1, &iValue2);

    Pattern(iValue1, iValue2);                              // Function call

    return 0;
}
